test/test_valgrind.sh

```diff
@@ -121,7 +121,10 @@ for test in $(ls -1 umf_test-*); do
 
 	[ "$FILTER" != "" ] && echo -n "($FILTER) "
 
+	LAST_FAILED=0
+
 	if ! HWLOC_CPUID_PATH=./cpuid valgrind $OPTION $OPT_SUP --gen-suppressions=all ./$test $FILTER >$LOG 2>&1; then
+		LAST_FAILED=1
 		FAIL=1
```

Comment:
perhaps we should be more descriptive with the `FAIL` variable name now - perhaps `ANY_TEST_FAILED` or something...?

Reply:
Done:
- `FAIL` replaced with `ANY_TEST_FAILED` and
- `LAST_FAILED` replaced with `LAST_TEST_FAILED`